packages/kbn-unified-data-table/src/utils/get_render_cell_popover.tsx

```diff
@@ -28,6 +28,7 @@ export const getCustomCellPopoverRenderer = () => {
     useEffect(() => {
       setCellPopoverProps({
         panelClassName: 'unifiedDataTable__cellPopover',
+        panelProps: { css: { maxInlineSize: 'min(75vw, 600px) !important' } },
```

Comment:
> [!NOTE]
> Force wider inline size on the popover panel to match the UI Design.

Reply:
I updated this part to make it wider only for an allowed list of columns. 
Not the ideal fix as we open the door to visual inconsistency, but it should work for this scenario.
[fix(discover): make popover wide for limited columns](https://github.com/elastic/kibana/pull/192567/commits/e87d6a8400f6931dac16e64c60b39bd58fd6f8a0)